src/rgw/rgw_rest_swift.cc

```diff
@@ -174,7 +174,7 @@ void RGWListBuckets_ObjStore_SWIFT::send_response_begin(bool has_buckets)
             user_quota,
             static_cast<RGWAccessControlPolicy_SWIFTAcct&>(*s->user_acl));
     dump_errno(s);
-    end_header(s, NULL, NULL, NO_CONTENT_LENGTH, true);
+    end_header(s, NULL, NULL, s->formatter->get_len(), true);
```

Comment:
Hmm, I bet that `formatter` is empty at this moment and call to `get_len()` would return `0`. Anyway, I don't think you want to specify any `Content-Length` here. Instead, just append the single `Accept-Ranges` header. :-)

```cpp
    dump_header(s, "Accept-Ranges", "bytes");
```

Reply:
@rzarzynski Thanks!
Yes, I want to add Accept-Ranges in header. :) 
I will add dump_header() before end_header() function, and update the patch. Thanks a lot! :)